crates/turborepo-lib/src/microfrontends.rs

```diff
@@ -20,7 +20,8 @@ pub struct MicrofrontendsConfigs {
 
 #[derive(Debug, Clone, Default, PartialEq)]
 struct ConfigInfo {
-    tasks: HashSet<TaskId<'static>>,
+    // A map from tasks declared in the configuration to the application that they belong to
+    tasks: HashMap<TaskId<'static>, String>,
```

Comment:
We need to store the application name as that is what the microfrontends local proxy uses to identify locally running applications.